### ISSUE TITLE
accessibility: `alt` texts for all images.  (missing on categories.  how do we grep for all instances of this problem?)

### ISSUE THREAD

**Issue by fisx** [liqd/aula#819]

**Comment by andorp** [comment]
Everywhere we use the `img_` combinator there should be an `alt` attribute attached.

**Comment by fisx** [comment]
category images don't need an alt tag because the `li_` has a text child.  same for many badges.
